devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy

```diff
@@ -205,19 +205,17 @@ def configure() {
 		return refresh() +
 			zigbee.temperatureConfig(30, 300) +
 			zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, DataType.UINT8, 30, 21600, 0x10) +
-			zigbee.configureReporting(0x0405, 0x0000, DataType.UINT16, 30, 3600, 100, [destEndpoint: 0x02])
+			zigbee.configureReporting(zigbee.RELATIVE_HUMIDITY_CLUSTER, 0x0000, DataType.UINT16, 30, 3600, 100, [destEndpoint: 0x02])
 	} else if (isFrientSensor()) {
 		return refresh() + 
 			zigbee.configureReporting(zigbee.RELATIVE_HUMIDITY_CLUSTER, 0x0000, DataType.UINT16, 60, 600, 1*100) +
 			zigbee.configureReporting(zigbee.TEMPERATURE_MEASUREMENT_CLUSTER, 0x0000, DataType.INT16, 60, 600, 0xA) +
 			zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0020, DataType.UINT8, 30, 21600, 0x1)
 	} else if (isEWeLinkTh01()) {
 		return refresh() +
-			zigbee.configureReporting(0xFC45, 0x0000, DataType.UINT16, 30, 3600, 100, ["mfgCode": 0x104E]) +   // New firmware
-			zigbee.configureReporting(0xFC45, 0x0000, DataType.UINT16, 30, 3600, 100, ["mfgCode": 0xC2DF]) +   // Original firmware
 			zigbee.batteryConfig() +
-			zigbee.temperatureConfig(3600, 7200) +
-			zigbee.configureReporting(0x0405, 0x0000, DataType.UINT16, 3600, 7200, null)
+			zigbee.temperatureConfig(30, 300, 50) +
```

Comment:
We don't typically specify the reportableChange argument. Any specific reason you did it here?

Reply:
@tpmanley Just to get half-degrees. More so for Celsius users (since 1C corresponds to > 1F). Probably fine at just 1 degree. I'm working with a community dev about the health of this device, which seems that it is a CST device.

Reply:
Consistency is probably a good idea. I believe this is a CST device, too.

Reply:
Yeah, let's go with consistency and 1 degree reportableChange which is the default. If it changes less than a degree it will still report within 5 minutes but this will potentially save a little battery if the value is changing more frequently.